hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -158,6 +161,8 @@
   private final RegionStates regionStates = new RegionStates();
   private final RegionStateStore regionStateStore;
 
+  private final ConcurrentMap<ServerName, Set<byte[]>> rsReports = new ConcurrentHashMap<>();
```

Comment:
It looks like all of our access is synchronized, why are we using CHM?

Reply:
Changed to HashMap in new patch.